already in upstream ports repo

From 146c8734c741e7f592cf07e7fd12644506c4c4c5 Mon Sep 17 00:00:00 2001
From: Aurelien Jarno <aurelien@aurel32.net>
Date: Tue, 1 Mar 2011 09:59:13 +1000
Subject: [PATCH] alpha: Define F_GET/SETPIPE_SZ

---
 ChangeLog.alpha                            |    5 +++++
 sysdeps/unix/sysv/linux/alpha/bits/fcntl.h |    2 ++
 2 files changed, 7 insertions(+), 0 deletions(-)

2011-02-28  Aurelien Jarno  <aurelien@aurel32.net>

	* sysdeps/unix/sysv/linux/alpha/bits/fcntl.h (F_SETPIPE_SZ,
	F_GETPIPE_SZ): Define.

diff --git a/sysdeps/unix/sysv/linux/alpha/bits/fcntl.h b/sysdeps/unix/sysv/linux/alpha/bits/fcntl.h
index 860e9ac..e5e726b 100644
--- ports/sysdeps/unix/sysv/linux/alpha/bits/fcntl.h
+++ ports/sysdeps/unix/sysv/linux/alpha/bits/fcntl.h
@@ -97,6 +97,8 @@
 # define F_SETLEASE	1024	/* Set a lease.	 */
 # define F_GETLEASE	1025	/* Enquire what lease is active.  */
 # define F_NOTIFY	1026	/* Request notfications on a directory.	 */
+# define F_SETPIPE_SZ	1031    /* Set pipe page size array.  */
+# define F_GETPIPE_SZ	1032    /* Set pipe page size array.  */
 #endif
 #ifdef __USE_XOPEN2K8
 # define F_DUPFD_CLOEXEC 1030	/* Duplicate file descriptor with
-- 
1.7.4.1

